README.md

```diff
@@ -501,7 +501,7 @@ v.validate({ mac: "01C8.954B.65FE" }, schema); // Valid
 v.validate({ mac: "01c8.954b.65fe", schema); // Valid
 v.validate({ mac: "01-C8-95-4B-65-FE" }, schema); // Valid
 v.validate({ mac: "01-c8-95-4b-65-fe" }, schema); // Valid
-v.validate({ mac: "01:C8:95:4B:65:FE" }, schema); // Fail
+v.validate({ mac: "01C8954B65FE" }, schema); // Fail
 ```
```

Comment:
It means `"01:C8:95:4B:65:FE"` is valid? If yes, please also add it as valid example.

Reply:
> It means `"01:C8:95:4B:65:FE"` is valid? If yes, please also add it as valid example.

it is the very first line, in this diff is not visible

Reply:
@icebob https://github.com/icebob/fastest-validator/commit/88675f8257386dac153df06c758e8362221029ab#diff-04c6e90faac2675aa89e2176d2eec7d8R498

Reply:
Sorry, yes, diff doesn't show it.